common/test/acceptance/tests/lms/test_lms_instructor_dashboard.py

```diff
@@ -361,6 +391,84 @@ def test_can_reset_attempts(self):
         exam_attempts_section.remove_student_attempt()
         self.assertFalse(exam_attempts_section.is_student_attempt_visible)
 
+    @attr('a11y')
+    def test_track_selection_a11y(self):
+        """
+        Proctored: Track selection accessibility tests
+        """
+        self.track_selection_page.a11y_audit.config.set_rules({
```

Comment:
OK, so I'm talking specifically about the a11y test points in this class. So anything in a different test class doesn't relate here. Let's look at test_track_selection_a11y-- you noted that the track selection "visit" method happens at L275. That's in the "_login_as_a_verified_user" helper method, so it only gets called if setUp or your test method calls something that calls that method. I don't believe that is the case for your a11y test method.

If this doesn't make sense, let's talk in person.